queries.sql

```diff
@@ -1,8 +1,54 @@
+/*Queries that provide answers to the questions from all projects.*/
+
 SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE neutered = true;
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
 SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE name != 'Gabumon';
\ No newline at end of file
+SELECT name FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* Query to update and read the animals table data */
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
```

Comment:
Very well done in making this project to this point. 

- [ ] You should have a query to verify that a change was made - `SELECT species from animals;` before and after ROLLBACK. Kindly add this.

Reply:
- [ ] @tanveerisonline   you haven't implemented the changes requested to have a query to verify changes made before and after rollback when updating the animals' species to unspecified. Kindly add the query and also add screenshots for the same.